target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -857,6 +857,20 @@ define Device/livebox
 endef
 TARGET_DEVICES += livebox
 
+### Netcomm ###
+define Device/NB604N
+  $(Device/bcm63xx)
+  IMAGES += sysupgrade.bin
+  DEVICE_VENDOR := Netcomm
+  DEVICE_MODEL := NB604N
+  DEVICE_DTS := nb604n
```

Comment:
Remove this line, and rename DTS file like soc-vendor-model.dts:
bcm6328-netcomm-nb604.dts